diff --git a/build.zig b/build.zig
index 25444c8..52bf952 100644
--- a/build.zig
+++ b/build.zig
@@ -21,6 +21,12 @@ pub fn build(b: *std.Build) void {
 
     lib.root_module.addImport("vapoursynth", vapoursynth_dep.module("vapoursynth"));
     lib.linkLibC();
+    lib.linkSystemLibrary("m");
+    lib.linkSystemLibrary("log");
+
+    if (target.result.isAndroid()) {
+        lib.addLibraryPath(.{ .cwd_relative = "/system/lib64" });
+    }
 
     if (lib.root_module.optimize == .ReleaseFast) {
         lib.root_module.strip = true;
